app/src/main/java/com/example/pinterbest/utilities/Validator.kt

```diff
@@ -1,10 +1,21 @@
 package com.example.pinterbest.utilities
 
+import android.util.Patterns
 import android.widget.EditText
+import com.google.android.material.textfield.TextInputLayout
 import java.util.regex.Pattern
 
 object Validator {
     private const val PASSWORD_MIN_SIZE = 8
+    private val PASSWORD_POLICY =
+        """Пароль должен состоять минимум из 8 символов, а также
+            |- должен содержать хотя бы одну заглавную букву,
+            |- хотя бы одну строчную букву,
+            |- хотя бы одну цифру и
+            |- хотя бы один специальный символ среди ~!@#$%^&*()-_=+|[]{};:'\",<.>/?""".trimMargin()
+
+    private const val NAME_VALIDATION_MSG = "Введите корректное имя"
+    private const val EMAIL_VALIDATION_MSG = "Введите корректный электронный адрес"
 
```

Comment:
nit: let's put it into android resources

Reply:
Fixed here https://github.com/enginire4ek/Pinterbest/pull/3/commits/f47063b96f0a24a56ad6def81896bbe4dee63167